lib/wibox/init.lua

```diff
@@ -347,10 +356,16 @@ local function new(args)
 
     -- If a value is not found, look in the drawin
     setmetatable(ret, {
-        __index = w,
+        __index = function(self, k)
+            if wibox["get_"..k] then
+                return self["get_"..k](self)
+            else
+                return w[k]
+            end
+        end,
         __newindex = function(self, k,v)
-            if wibox["set_"..k] then
-                wibox["set_"..k](v)
+            if rawget(self, "set_"..k) then
+                self["set_"..k](self, v)
```

Comment:
This seems like a fixup for the commit that added this `wibox["set_"..k]`, no? Perhaps this part of the change should be squashed into that.

(Why does this need/use rawget?)